modules/core/src/test/java/org/apache/ignite/internal/mxbean/IgniteStandardMXBeanTest.java

```diff
@@ -0,0 +1,576 @@
+        getParameterNameWithMethodNameAndParamIndex(TEST_METHOD_7, ZERO_INDEX);
+    }
+
+    /**
+     * A test method that represents a situation in which the method has both old and new annotations.
+     * All annotations parameters are valid.
+     *
+     * @throws NoSuchMethodException if method is not found.
+     */
+    @Test
+    public void getParameterName_BothOldAndNewAnnotations() throws NoSuchMethodException {
+        String actualResult = getParameterNameWithMethodNameAndParamIndex(TEST_METHOD_9, FIRST_INDEX);
+
+        assertEquals(SECOND_NAME_METHOD_ANNOTATION, actualResult);
+    }
+
+    /**
+     * A test method that represents a situation in which the method has no annotations at all.
+     *
+     * @throws NoSuchMethodException if method is not found.
+     */
+    @Test
+    public void getParameterName_NoAnnotations() throws NoSuchMethodException {
+        String actualResult = getParameterNameWithMethodNameAndParamIndex(TEST_METHOD_10, FIRST_INDEX);
+
+        assertEquals(NAME, actualResult);
+    }
+
+    /**
+     * Utility method that returns the description for method argument by method name and parameter index.
+     *
+     * @param methodName method name from interface TestInterface.
+     * @param paramIndex the sequence number of the argument considered.
+     * @return the description for method argument.
+     * @throws NoSuchMethodException if a matching method is not found.
+     */
+    private String getDescriptionWithMethodNameAndParamIndex(String methodName, int paramIndex)
+        throws NoSuchMethodException {
+        MBeanOperationInfo operationInfo = getMBeanOperationInfoWithMehtodName(methodName);
+        return igniteStandardMXBean.getDescription(operationInfo, parameterInfo, paramIndex);
+    }
+
+    /**
+     * Utility method that returns the name for method argument by method name and parameter index.
+     *
+     * @param methodName method name from interface TestInterface.
+     * @param paramIndex the sequence number of the argument considered.
+     * @return the name for method argument.
+     * @throws NoSuchMethodException if a matching method is not found.
+     */
+    private String getParameterNameWithMethodNameAndParamIndex(String methodName, int paramIndex)
+        throws NoSuchMethodException {
+        MBeanOperationInfo operationInfo = getMBeanOperationInfoWithMehtodName(methodName);
+        return igniteStandardMXBean.getParameterName(operationInfo, parameterInfo, paramIndex);
+    }
+
+    /**
+     * Utility method for getting instance of MBeanOperationInfo constructed with default description from TestInterface
+     * by its method name.
+     *
+     * @param methodName method name from interface TestInterface.
+     * @return MBeanOperationInfo.
+     * @throws NoSuchMethodException if a matching method is not found.
+     */
+    private MBeanOperationInfo getMBeanOperationInfoWithMehtodName(String methodName) throws NoSuchMethodException {
```

Comment:
Abbreviation should be used for parameter names `method` -> `m`. See abbreviation rules.

Reply:
Fixed.